src/molecules/Select/index.tsx

```diff
@@ -24,6 +24,7 @@ export interface SelectProps<T> extends OptionsListProps<T> {
   hasError?: boolean;
   invertColor?: boolean;
   renderLeftIcon?: React.ReactNode;
+  tabIndex?: string;
```

Comment:
should the `tabIndex` prop get passed down to the `<Input />` component? You could destructure it right after L62 and it doesn't need a default value since it's also optional in the `Input`